src/components/freshness-bar/stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+
+import { addMonths } from 'date-fns';
+import { FreshnessBar, Props } from './FreshnessBar';
+import Label from '../label';
+import { Flex } from '../flex';
+
+const monthAgo = addMonths(new Date(), -1);
+const halfAYearAgo = addMonths(new Date(), -6);
+const threeMonthsAgo = addMonths(new Date(), -3);
+const almostAYearAgo = addMonths(new Date(), -11);
+
+export default {
+  title: 'Quartz/FreshnessBar',
+  component: FreshnessBar,
+  args: {
+    expirationDate: new Date(monthAgo),
+    size: 'md',
+  },
+  argTypes: {
+    expirationDate: {
+      control: {
+        type: 'date',
+      },
+    },
+  },
```

Comment:
the addon-docs doesn't pick the property type correctly in case of Date?

Reply:
yeah, it gives a timestamp input
![image](https://user-images.githubusercontent.com/19791699/184352324-c927b930-7324-407a-bd71-41cc33125b06.png)